### DISCUSSION TITLE
@$(which ls) from tutorial not working

### DISCUSSION THREAD

**Discussion by aspieln3r** [xonsh/xonsh#4471] [Q&A]
from https://xon.sh/tutorial.html#command-substitution-with
```
 # this does the same thing, but is much more concise
 @$(which ls)
some_file  some_other_file
```
When I run this, I get error like this
```
uname@hname $ @$(which ls)
xonsh: subprocess mode: command is empty
```
running `which ls` normally returns
 `ls --color=auto -v`

**Comment by aspieln3r** [comment]
I found that doing like this works
@(list($(echo ls -al).split()))
so the output needs to be a list of strings. Maybe tutorial is wrong?

**Comment by agoose77** [comment]
The tutorial command works as expected for me. Can you reproduce this with the latest xonsh (0.10.1)?
